apps/prairielearn/src/lib/db-types.ts

```diff
@@ -236,7 +236,7 @@ export const AiGradingJobSchema = z.object({
   id: IdSchema,
   job_sequence_id: IdSchema.nullable(),
   model: z.string(),
-  prompt: z.any(),
+  prompt: z.unknown(),
```

Comment:
IMO this is a better option than `z.any()` if we don't actually know the shape, this will force us to navigate into the value carefully.